### PR TITLE
Clair: handle messages from Clair v4

### PR DESCRIPTION
**Pull request by Allda**
In Clair v4 a message format changed a bit. NVRA is no longer available
and it was replaced by image_id.

This commit also unifies strings in Clair related files. All strings
uses double quotes now.

### CODE REVIEW

File: fedmsg_meta_umb/clair.py [hunk 1]
```diff
@@ -20,82 +20,92 @@
...
-                template = self._('Scan request: {brew_build}.{architecture}')
-            elif inner_msg.get('imageDigest'):
-                template = self._('Scan request: {imageDigest}')
+                template = self._("New notification {Notification[Name]}")
+        elif topic.endswith("clair.scan"):
+            if inner_msg.get("nvr") and inner_msg.get("arch"):
+                template = self._(
+                    "New image scanned: {nvr}.{arch} ({grades[0][grade]})"
+                )
+            elif inner_msg.get("image_id"):
```
Comment (mikebonnet) [inline fedmsg_meta_umb/clair.py:57]: In what cases is `nvra` present? Why not check and display that, before defaulting to the `image_id`?

File: fedmsg_meta_umb/clair.py [hunk 2]
```diff
@@ -20,82 +20,92 @@
...
     def packages(self, msg, **config):
-        inner_msg = msg['msg']
-        if msg['topic'].endswith('clair.scan'):
-            return set([inner_msg['nvr'].rsplit('-', 2)[0]])
-        elif msg['topic'].endswith('clair.command'):
-            if inner_msg.get('brew_build'):
-                return set([inner_msg['brew_build'].rsplit('-', 2)[0]])
+        inner_msg = msg["msg"]
+        if msg["topic"].endswith("clair.scan") and inner_msg.get("nvr"):
+            return set([inner_msg["nvr"].rsplit("-", 2)[0]])
```
Comment (mikebonnet) [inline fedmsg_meta_umb/clair.py:101]: You could parse a package name out of `nvra` in the same way.

**Review by mikebonnet — CHANGES_REQUESTED** [review]
The new message format which contains a `nvra` (NVR + arch) could be better supported.

**Comment by Allda** [comment]
NVRA is only available in Clair v4 which was deployed only to dev environment. Stage and prod are going to be released in the next weeks. Should I wait for all environment and then remove Clair v3 related code and replace it with the only v4? The current PR supports both versions.

**Comment by mikebonnet** [comment]
> NVRA is only available in Clair v4 which was deployed only to dev environment. Stage and prod are going to be released in the next weeks. Should I wait for all environment and then remove Clair v3 related code and replace it with the only v4? The current PR supports both versions.

Supporting both formats is good (messages in the old format will still be available in the database), and I don't think there's any need to wait for deployments. My comments just meant that anywhere you're using `{nvr}.{arch}` with v3 messages, you could be using `{nvra}` with v4 messages, which would be more informative then displaying an `image_id`.

**Comment by mikebonnet** [comment]
Could you rebase this?

**Comment by Allda** [comment]
@mikebonnet sorry that it took me so long but I kind of forget about this PR. I added changes based on your comments, added a new message type, and rebased the PR.
Travis so very slow today so hopefully checks will pass soon.

**Comment by mikebonnet** [comment]
This looks great, thanks!

**Pull request merged**